config/locales/en.yml

```
@@ -1972,7 +1972,7 @@ en:
         contributors_footer_1_html: |
           For further details of these, and other sources that have been used
           to help improve OpenHistoricalMap, please see the <a
-          href="https://wiki.openstreetmap.org/wiki/Open_Historical_Map/OHM_Contributors">Contributors
+          href="https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/OHM_Contributors">Contributors
           page</a> on the OpenHistoricalMap Wiki.
```

We don’t have our own wiki yet: #561.

```suggestion
          page</a> on the OpenStreetMap Wiki.
```